libavb_ab/avb_ab_flow.c

```diff
@@ -443,6 +443,24 @@ AvbIOResult avb_ab_mark_slot_active(AvbABOps* ab_ops,
   return ret;
 }
 
+unsigned int avb_ab_get_active_slot(AvbABOps* ab_ops) {
+  AvbABData ab_data, ab_data_orig;
+  AvbIOResult ret;
+
+  ret = load_metadata(ab_ops, &ab_data, &ab_data_orig);
+  if (ret != AVB_IO_RESULT_OK) {
+    goto out;
+  }
+
+  for (unsigned int i = 0; i < 2; i++) {
+      if (ab_data_orig.slots[i].priority == AVB_AB_MAX_PRIORITY)
+          return i;
+  }
+
+out:
+  return 0;
```

Comment:
in line457, if return i=0 means, it get active_slot 0. In line 461 error case, return 0 also means get active slot 0? Is it as expected？

Reply:
Yes, it's intended to do this. If some error happens, then return the first slot. Do you have any suggestion for this?